schema/dataKeys.json

```diff
@@ -584,6 +585,51 @@
         }
       }
     },
+    "ootsign": {
```

Comment:
these sound like they should be per component

Reply:
I see it at the same level as other objects in the repo, could you please clarify? Sure I'm missing it but I could not see it.

Reply:
We are not having multiple components per application  but one application/on component per vendor.

Reply:
They are still relevant to the component only, so I think they should be nested accordingly. It is confusing to leave it like this. How does someone who looks at the data file and doesn't know the out-of-tree stuff well supposed to know these fields relate to the component?